library/ws-widget/collection/src/lib/btn-page-back/btn-page-back.component.ts

```diff
@@ -1,18 +1,19 @@
 import { animate, style, transition, trigger } from '@angular/animations'
 import { Component, HostBinding, Input, OnInit } from '@angular/core'
 import { Router } from '@angular/router'
-// import { environment } from './../../../environments/environment'
+
 import { NsWidgetResolver, WidgetBaseComponent } from '@sunbird-cb/resolver'
 import { ConfigurationsService, MultilingualTranslationsService, NsInstanceConfig } from '@sunbird-cb/utils'
+
```

Comment:
please remove extra line spacing, also do documentation for imported librarry by segregating in respect to  from angular, service files etc.